### PR TITLE
request additional OIDC scopes (SC-260)

### PR DESCRIPTION
**Pull request by zimingd**
The additional scopes are necessary for applications behind the load balancer(e.g. R Studio Server w/ preinstalled Synapse client) to use the token andinteract with Synapse.

### CODE REVIEW

File: ruler/app.py [hunk 1]
```diff
@@ -137,6 +137,7 @@ def create(event, context):
             "UserInfoEndpoint": oidc_user_info_endpoint,
             "ClientId": oidc_client_id,
             "ClientSecret": client_secret,
+            "Scope": "openid view download modify",
```
Comment (ahayden) [inline ruler/app.py:140]: Previously this was undefined, which is assumed to be the `openid` scope only, IIRC, which is only identity

**Review by ahayden — COMMENTED** [review]
This new scope allows view/download/modify on all of the user's Synapse data, right? Is it possible to scope to specific data sets?

**Review by zaro0508 — APPROVED** [review]

File: ruler/app.py (hunk 1 above)
Comment (zimingd) [reply ruler/app.py:140]: Yep. Documentation:
https://docs.aws.amazon.com/elasticloadbalancing/latest/APIReference/API_AuthenticateOidcActionConfig.html

Comment (zimingd) [reply ruler/app.py:140]: > This new scope allows view/download/modify on all of the user's Synapse data, right? Is it possible to scope to specific data sets?

I don't believe we currently support that in Synapse.

Comment (brucehoff) [reply ruler/app.py:140]: OAuth scope does not let us specific specific data sets or projects. (There's a reason for this!) The listed scopes are correct. In particular they omit `offline_access` and `authorize` which is correct.  The complete list is here: https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/oauth/OAuthScope.html

**Review by brucehoff — APPROVED** [review]

**Pull request merged**